api_app/tests_ma/test_api/test_routes/test_workspaces.py

```diff
@@ -106,6 +107,17 @@ def sample_workspace_service(workspace_service_id=SERVICE_ID, workspace_id=WORKS
     )
 
 
+def sample_deployed_workspace_service(workspace_service_id=SERVICE_ID, workspace_id=WORKSPACE_ID):
```

Comment:
as above, patching shouldn't require a deployed workspace service, so can probably make do with a sample workspace service if it is needed at all -- in reality patching should not need the workspace service -- the chain is verified in the PR I have open by just getting the resource by full ID